### PR TITLE
React capstone project - Crypto Watcher

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
### Project Requierements - React App ( Crypto Watcher )

- Choose API : [Coin Stats](https://documenter.getpostman.com/view/5734027/RzZ6Hzr3?version=latest) ✅
- Design : [Guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) ✅
- Interactions 
  - Home page ✅
  - Details page ✅
- Testing requirements
  - Create unit tests for pure functions in your Redux code. ✅
  - Create integration tests for your application using the React Testing Library. ✅
- Technical requirements
  - The project is a single-page application (SPA) built with React and Redux. ✅
  - The data retrieved from the API should be stored in the Redux store. ✅
  - You should filter the data that you retrieve from the API using a Filter stateless component. ✅
  - Every page (the main page and the pages for each item) should have a unique route within the SPA. ✅
  - The project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/). ✅
- Project documentation
  - Description of the project. ✅
  - Live Demo of the project. ✅
  - You should also highlight some interesting piece of code or something you built that you are very proud of. ✅
- All linters have been checked! ✅
---
- Loom Video: [React Capstone Project: Crypto Watcher Presentation](https://www.loom.com/share/3679ac9211914b9abdf5c5f87bf78a18?sid=81ad2818-91a8-46ec-a5ca-0f97aa6cb52f)

### CODE REVIEW

File: src/tests/__snapshots__/navbar.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (Alfredbis29) [inline src/tests/__snapshots__/navbar.test.js.snap:1]: - [ ] I can see you made a reasonable effort on this project but I've reviewed the issue related to Jest encountering an "unexpected token" error. I want to highlight that this error typically occurs due to non-standard JavaScript syntax or misconfigured Jest settings.

Please revisit your code and Jest configuration to identify and address these issues. Feel free to ask for assistance if you have any questions or require clarification on specific points.
 check the screenshot below and kindly fix this :wink:  **Your Test are not Passing**

Screenshot from 2023-09-09 04-01-54

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @JuanDiegoGuerra 
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
:heavy_check_mark: You have a good code structure.
:heavy_check_mark: Your project looks professional
 
 ## Required Changes 

- [ ] Check Under the Review

 ## Optional suggestions 

- [ ] N/A 

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JuanDiegoGuerra** [comment]
###  Required changes made 🛠
**All linters are enable!**
- Fixed: _"Please revisit your code and Jest configuration to identify and address these issues. Feel free to ask for assistance if you have any questions or require clarification on specific points.
check the screenshot below and kindly fix this 😉 Your Test are not Passing"_ ✅
  - Now the tests are passing correctly! ✅
  - image


todolist1

**Review by OLIPLICHE — APPROVED** [review]
Hi @JuanDiegoGuerra, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
-  App has unit tests for several components of the app(✅)) 👏
- App has integration tests 👏
- The test has passed 👏 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**